additional_recipes/ros-distro-mutex/recipe.yaml

```diff
@@ -29,11 +29,11 @@ requirements:
   # if the upstream package does not have run_exports
   # please change it in the conda_build_config.yaml!
   run_constrained:
-    - boost-cpp 1.78
-    - pcl 1.12
+    - boost-cpp 1.82
```

Comment:
Should we use libboost now, after the renaming? More in general, I am not sure if this run_constrained deps in the muted are actually useful, they created problems to me in the past as I have a package that depends on PCL, and I want to install it with some Ros packages that do not depend on PCL, and I still can't due to this run_constrained dep on the mutex. Anyhow no need to block the rebuild on this if it is working.

Reply:
That’s a good call. Now that we depend on libboost which has run exports we should not need this anymore.